### ISSUE TITLE
AndroidJUnit4 is deprecated

### ISSUE THREAD

**Issue by ykws** [ykws/test-driven-development-by-example-kotlin#1]

**Comment by ykws** [comment]
https://stackoverflow.com/questions/52776716/androidjunit4-class-is-deprecated-how-to-use-androidx-test-ext-junit-runners-an

[Android Doc](https://developer.android.com/reference/androidx/test/runner/AndroidJUnit4) says -

> This class is deprecated.
> 
> use androidx.test.ext.junit.runners.AndroidJUnit4 instead.

So replace following line -

```
androidTestImplementation 'androidx.test:runner:1.1.0'
```

with -

```
androidTestImplementation "androidx.test.ext:junit:1.0.0"
```

**Comment by ykws** [comment]
fixes d6ac760770a348ec5813bb24e30dab7614dad06e
